google/cloud/firestore_v1/watch.py

```diff
@@ -569,7 +568,9 @@ def push(self, read_time, next_resume_token):
             self._snapshot_callback(
                 keys,
                 appliedChanges,
-                datetime.datetime.fromtimestamp(read_time.seconds, pytz.utc),
+                read_time
+                # TODO(microgen): now a datetime
+                # datetime.datetime.fromtimestamp(read_time.seconds, pytz.utc),
```

Comment:
Is there an issue with the datetime change here? The type passed to `_snapshot_callback` should be the same. Do we lose precision?